Sources/SwiftDocC/Infrastructure/DocumentationContext.swift

```diff
@@ -612,6 +612,17 @@ public class DocumentationContext {
                 resolver.visit(documentationNode.semantic)
             }
             
+            // Also resolve the node's alternate representations. This isn't part of the node's 'semantic' value (resolved above).
+            documentationNode.metadata?.alternateRepresentations.forEach { alternateRepresentation in
```

Comment:
style: I'll just mention this once but I feel like `forEach` is just a worse and less capable version of a regular for loop and there's practically no use for it ever.